docs/attributes-registry/nodejs.md

```diff
@@ -0,0 +1,15 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Nodejs
```

Comment:
I think it makes sense to add this to the abbreviations list such as NodeJS to the weaver.yaml

Reply:
Ah cool! I wasn't aware of it, and it looks a lot better now!